Mathlib/CategoryTheory/Limits/IndYoneda.lean

```diff
@@ -198,6 +198,35 @@ lemma colimitCoyonedaHomIsoLimit_π_apply (f : colimit (D.rightOp ⋙ coyoneda)
 
 end ProCoyonedaContravariant
 
+section ProCoyonedaContravariantRightOp
+
+variable (D : I ⥤ Cᵒᵖ) (F : C ⥤ Type u₂)
+variable [HasColimit (D ⋙ coyoneda)] [HasLimitsOfShape Iᵒᵖ (Type (max u₁ u₂))]
+
+/-- Pro-Coyoneda lemma: morphisms from colimit of coyoneda of diagram `D` to `F` is limit
+of `F` evaluated at `D`. This variant is for contravariant diagrams, see
+`colimitCoyonedaHomIsoLimit'` for a covariant version. -/
+noncomputable def colimitCoyonedaHomIsoLimitLeftOp :
+    (colimit (D ⋙ coyoneda) ⟶ F) ≅ limit (D.leftOp ⋙ F ⋙ uliftFunctor.{u₁}) :=
+  colimitHomIsoLimitYoneda _ F ≪≫
+    HasLimit.isoOfNatIso (isoWhiskerLeft (D.leftOp ⋙ Prod.sectl C F) (coyonedaLemma C))
```

Comment:
If you add a
```
local instance : HasColimit (D.leftOp.rightOp ⋙ coyoneda) :=
  inferInstanceAs <| HasColimit (D ⋙ coyoneda)
```
this can become
```suggestion
  colimitCoyonedaHomIsoLimit D.leftOp F
```

Reply:
I have applied this change, but I chose to inline the instances into the terms instead of adding a `local instance`. This way, it is clear that the local instance is not required in the statement of the definition/theorems, which is important to make sure that users can use these results without having to restate the local instances.